src/Server.h

```diff
@@ -70,6 +70,14 @@ class cServer
 	size_t GetNumPlayers(void) const { return m_PlayerCount; }
 	void SetMaxPlayers(size_t a_MaxPlayers) { m_MaxPlayers = a_MaxPlayers; }
 
+	// tolua_end
+	/** Add a Forge mod to the server ping list. */
+	bool RegisterForgeMod(AString & a_ModName, AString & a_ModVersion, UInt32 a_ProtocolVersionNumber);
+	// tolua_begin
```

Comment:
Separate the `tolua` comments with empty lines both above and below the comment, to make it stand out more.